notificationdemo/source/objc/NotificationDemo/ViewController.m

```diff
@@ -143,12 +158,19 @@ - (NSArray *)extractOptionalTopicIds:(NSArray *)topics {
     return topicIds;
 }
 
-- (void)addLogWithText:(NSString *) text {
+- (void)addLogWithText:(NSString *)text textColor:(UIColor *)textColor {
```

Comment:
AFAIK, u need to add new ```(void)addLogWithText:(NSString *)text``` method without *textColor* arg, where you will call ```(void)addLogWithText:(NSString *)text textColor:(UIColor *)textColor``` method with second arg as nil.
Actually, it will protect you from calling method with nil arg everywhere.